Libraries/Connect/Interfaces/ConnectError.swift

```diff
@@ -73,6 +73,23 @@ public struct ConnectError: Swift.Error, Sendable {
             case payload = "value"
         }
 
+        public init(from decoder: Swift.Decoder) throws {
+            let container = try decoder.container(keyedBy: CodingKeys.self)
+
+            // Read the base64-encoded payload and then pad if needed
+            let encodedPayload = try container.decode(String.self, forKey: .payload)
+            let padded = encodedPayload.padding(
+                toLength: ((encodedPayload.count+3)/4)*4,
```

Comment:
```suggestion
                toLength: ((encodedPayload.count + 3) / 4) * 4,
```
Also, this is a bit magic -- can you inline an explanation for this?